### PR TITLE
Badge bugfix

### PR DESCRIPTION
**Pull request by tmathern**
## Issue

Fix N/A

## 🔗 Test URLs:

- Before: https://main--sgws-creative--hlxsites.hlx.page/bloominbrands/p/3-tmn-draft
- After: https://issue-95--sgws-creative--hlxsites.hlx.page/bloominbrands/p/3-tmn-draft

## 📝 Description:
- Fix badge overlapping with "pairs with" content

### CODE REVIEW

File: blocks/slides/slides.css [hunk 1]
```diff
@@ -205,6 +205,10 @@
   padding: var(--layout-size-xl) 0 0;
 }
 
+.slides .slide .pairs-with-text-badge {
```
Comment (tmathern) [inline blocks/slides/slides.css:208]: That's a bugfix

File: blocks/tabview/tabview.js [hunk 2]
```diff
@@ -66,7 +66,10 @@ async function loadTabPanel(panel) {
     const closeProgramButton = closeProgramView.querySelector('button');
     const programSlidesWrapper = panel.querySelector('.slides-wrapper.program-content');
     programSlidesWrapper.parentNode.insertBefore(closeProgramView, programSlidesWrapper);
-
+    const nestedSlides = programButton.previousElementSibling;
```
Comment (tmathern) [inline blocks/tabview/tabview.js:69]: That's a bugfix

File: blocks/video/video.js [hunk 3]
```diff
@@ -1,13 +1,12 @@
 import { createTag } from '../../scripts/scripts.js';
-import { createOptimizedPicture } from '../../scripts/lib-franklin.js';
```
Comment (tmathern) [inline blocks/video/video.js:2]: Not needed anymore

**Review by allentsliu — APPROVED** [review]
LGTM!

**Pull request merged**